### ISSUE TITLE
Error attempting to rejoin footnotes

### ISSUE THREAD

**Issue by wf49670** [davem2/dp2ppgen#1]
Using this input file: https://dl.dropboxusercontent.com/u/3229599/cicerov2-orig.txt

I get the following error when running with -pcfjs:

> ERROR: Attempt to join footnote failed!
> ERROR: ScanPg 241.png Footnote 411 (10585): *[Footnote: for his claim to keep his province is not more unconstitutional than his
> ERROR: ScanPg 242.png Footnote 412 (10627): [Footnote 1: The <i>Compitalia</i>, were always about this time, though the exact day

The footnote on 241.png should be joined to the footnote on 240.png, but isn't.

I'm not sure why the error refers to 242.png.

**Comment by davem2** [comment]
Looks like this markup in [Footnote 1] on scan page 240 is causing issues:

```
/*
[Greek: on men dê tode meixon epi kakon ê ote Kuklôps
eilei eni skêi glaphyrps kraterêphi biêphi.]
*/
```

If I remove those lines from the footnote (and fix some proofing errors listed below), then the footnotes are processed ok. 

```
--310.png
[Footnote 2 See p. 279.]
[Footnote 2: See p. 279.]

--332.png
that I shall be coming to the city walls[l] before long, that I
that I shall be coming to the city walls[1] before long, that I

--336.png
Missing [Footnote 2:]

--359:
done much less than he threatened, is regarded with affection;
done much less than he threatened, is regarded with affection;[1]
```

I'll look into why the /\* [Greek] */ is not being processed correctly.

**Comment by wf49670** [comment]
On 3/16/2015 7:16 PM, davem2 wrote:

> Looks like this markup in [Footnote 1] on scan page 240 is causing issues:
> 
> |/*
> [Greek: on men dê tode meixon epi kakon ê ote Kuklôps
> eilei eni skêi glaphyrps kraterêphi biêphi.]
> */
> |
> 
> If I remove those lines from the footnote (and fix some proofing 
> errors listed below), then the footnotes are processed ok.
> 
> |--310.png
> [Footnote 2 See p. 279.]
> [Footnote 2: See p. 279.]
> 
> --332.png
> that I shall be coming to the city walls[l] before long, that I
> that I shall be coming to the city walls[1] before long, that I
> 
> --336.png
> Missing [Footnote 2:]
> 
> --359:
> done much less than he threatened, is regarded with affection;
> done much less than he threatened, is regarded with affection;[1]
> |
> 
> I'll look into why the //[Greek]// is not being processed correctly.
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/davem2/dp2ppgen/issues/1#issuecomment-81983839.
> 
> Thanks. I thought I had fixed those proofing/formatting errors, but 
> perhaps not in the version of the file I sent.

Regars,
Walt

**Comment by davem2** [comment]
Fixed in e33d912
